src/algorithms.jl

```diff
@@ -67,6 +100,23 @@ See [`SumOfSquares.jl`](https://github.com/jump-dev/SumOfSquares.jl) for more de
 To use this solver, you need to import `SumOfSquares.jl` and a backend.
 Since the optimization problem is solved numerically and not with interval arithmetic, the
 result of this algorithm is not rigorous.
+
+```julia
+julia> using SumOfSquares, SDPA
+
+julia> backend = SDPA.Optimizer
+
+julia> enclose(x -> -x^3/6 + 5x, 1..4, SumOfSquaresEnclosure(; backend=backend))
+[4.8333, 10.541]
+```
+
+You can also pass additional keyword arguments that will be passed to the SOSMOdel, e.g. to
+print in non-verbose mode
+
+```julia
+julia> enclose(x -> -x^3/6 + 5x, 1..4, SumOfSquaresEnclosure(; backend=backend); QUIET=true)
+[4.8333, 10.541]
```

Comment:
At the end of the day it does not matter, but should we use the same example as in the other cases for a quick sense of comparison?
(I can run it in v1.6 if that is the problem.)

Reply:
I think it might be nice to have the same example everywhere, the only reason why I didn't use the same of the others is indeed because I don't have v1.6 on my pc

Reply:
```julia
julia> enclose(x -> 1 - x^4 + x^5, 0..1, SumOfSquaresEnclosure(; backend=backend))
Strange behavior : primal < dual :: line 144 in sdpa_solve.cpp
Strange behavior : primal < dual :: line 144 in sdpa_solve.cpp
[0.91808, 1.00001]
```

Reply:
The second call raises an error saying that `QUIET` is not a valid `JuMP` keyword argument.